app/models/user.rb

```diff
@@ -54,7 +54,7 @@ def create_personal_namespace!
     Namespace.find_or_create_by!(
       team:     team,
       name:     username,
-      registry: Registry.last # TODO: fix once we handle more registries
+      registry: Registry.get
```

Comment:
Please, don't remove the comment. Even if it's in the documentation of the `Registry.get` method, this might be useful when refactoring into having multiple registries.